src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NET6_0
```

Comment:
I suppose we should start the ifdef pattern here even though we don't technically need it in the first release. I assume we want an error to remind ourselves to update this file.

Reply:
What is the error for?

Reply:
Remind ourselves to update the ifdefs so each TFM has a new section. I'm hoping somewhere in our test pipeline this error will show up when we update the TFM.

Reply:
do you want to say net60 or greater? It's fairly unlikely this list is going to change very often, no?

Reply:
Exactly

Reply:
FYI, the conditionality of these usings are now specified in the target files with conditions like `'$(TargetFrameworkIdentifier)' == '.NETCoreApp'
                      and $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '6.0'))`